src/playground-layout.css

```
@@ -0,0 +1,113 @@
+  transition:
+          opacity 0.5s,
+          transform 0.25s ease;
+  overscroll-behavior: contain;
+}
+
+.OASidebar.open {
+  opacity: 1;
+  visibility: visible;
+  transform: translateX(0);
+  transition:
+          opacity 0.25s,
```

ESLint

[error] Delete `······`

(format/prettier)